src/System.Runtime.CompilerServices.Unsafe/dir.props

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.4.1</AssemblyVersion>
-    <PackageVersion>4.5.2</PackageVersion>
+    <PackageVersion>4.5.3</PackageVersion>
```

Comment:
Small note that we still build System.Runtime.CompilerServices.Unsafe on newer branches, but this is a dependency of System.Memory which we now only build on release/2.1, so I had to fix Unsafe here as well in order to ensure that latest version of System.Memory would be able to be consumed on .NET Framework facade-free.